src/main/webapp/WEB-INF/jsp/sicknote/sick_note_form.jsp

```diff
@@ -239,6 +239,36 @@
                     </div>
                 </div>
 
+                <div class="form-section">
+                    <div class="col-xs-12">
+                        <legend>
+                            <spring:message code="sicknote.data.furtherInformation.title"/>
+                        </legend>
```

Comment:
muss `legend` nicht innerhalb eines `forms` sein?

Reply:
in einem form ist es, aber Du meinst wahrscheinlich in einem fieldset (https://www.w3schools.com/tags/tag_fieldset.asp). Das wurde bisher nirgends so gemacht, könnte man aber natürlich anfangen. Würde das aber separat machen, oder wie siehst Du das?

Reply:
> Du meinst wahrscheinlich in einem fieldset 

nein, ich meinte `form` :)
hatte unten statt `form:foo` nur das `form` gelesen. daher...

alles gut